src/backend/chat/commands/builtin/quotes.ts

```diff
@@ -278,6 +278,11 @@ export const QuotesManagementSystemCommand: SystemCommand<{
                     const channelData = await TwitchApi.channels.getChannelInformation();
 
                     const currentGameName = channelData && channelData.gameName ? channelData.gameName : "Unknown game";
+                    
+                    //If no @ is included in the originator arg, set to @streamerName and the rest is the quote
+                    if (!args[1].includes("@")) { 
+                        args.splice(1,0,"@"+channelData.displayName) 
```

Comment:
Super minor thing, but string templates are generally preferred over string concatination:
```js
`@${channelData.displayName}`
```

Reply:
Addressed in 1f79440d57a7cf0a5673f68112b9311ea1dde902